### ISSUE TITLE
Add SageMath installation instructions

### ISSUE THREAD

**Issue by olexandr-konovalov** [ccp-codima/2020-02-20-lancaster#3]
@fredstro @slel I have populated the template with information - it's not final - we can update details later, but something that we can check and provide to @NadiaMazza to start to advertise it. Please check that it looks OK.

One of immediate TODOs is to write installation instructions for SageMath for all three systems. I've started that on 092ffd9cc76be7aa01f7a50b864d113d9a98e9e7 to minimally move away from Python instructions - will you be able to take over and complete it with proper SageMath ones, please?

**Comment by fredstro** [comment]
I wonder if it is more appropriate to simply add a link to the official instructions: http://doc.sagemath.org/html/en/installation/index.html
since it mentions all the different possibilities (e.g. conda-forge / docker / source / binary) 
Otherwise I would just copy - paste from there anyway. 

I am also not sure which installation type we should recommend. I think the binary is probably easiest on Linux and Mac but maybe conda-forge or docker are easier on Windows?

For the last few years I have only used source installation on Mac and Linux but it might be good to try the conda-forge and docker installations as well. Unfortunately I don't have any Windows machine to try on.

I am completely swamped with admin duties this week but I can try to write the mac and linux instructions (or copy the link) in the weekend.
If no one else has a Windows machine to try out on I can also try to ask around and see if I can find someone with a successful windows installation.

**Comment by olexandr-konovalov** [comment]
As a quick fix, a link to http://doc.sagemath.org/html/en/installation/index.html will work. 
It may be still be useful to provide a TL;DR version of them, with pointers to the way that you'd recommend to novices, who may find the long page with different options not helpful. For GAP, for example, this is done instructions are at https://carpentries-incubator.github.io/gap-lesson/setup/.

I support binary for Linux and Mac, but for Windows I suggest to test https://github.com/sagemath/sage-windows/releases by @embray which may be the best option.

**Comment by slel** [comment]
Linking to the official instructions sounds good, as well
as a short recap and recommendations for novices.

Ideally, find a few volunteers around you to try them out
and report on anything that needs clarifying or updating.

The Conda port of SageMath currently only works on Linux
and macOS, not Windows.

On Windows the Sage-Windows installer works very well
and is the easiest solution. This is also what one gets as
binaries for Windows from the Sage download page.

For more developer-minded Windows users, installing
Cygwin and building Sage from source also works; the
instructions for that are at

https://trac.sagemath.org/wiki/Cygwin64Port

**Comment by olexandr-konovalov** [comment]
Thanks @fredstro @slel. I've made a quick update at 333811c9ecc0ae6ae7d2f74ed52b8b80b74e90f0, so at least it's points to the right place, and we can now take our time, test instructions and improve setup instructions. Note that we want a simple process that works for new users, possibly never installing SageMath before, and will ask them to install it in advance, following instructions, so unlikely we will advise to use Cygwin64Port link, but we can mention it somewhere in the training materials (e.g. under "Extras" at http://alex-konovalov.github.io/sage-lesson/).

**Comment by embray** [comment]
> so unlikely we will advise to use Cygwin64Port link

Indeed, that's really unlikely to be useful to anyone but developers. 

It may also be worth noting that by installing Sage for Windows they also get a reasonably well-functioning GAP along with it (although only with a minimal set of packages--for more packages they need to install the `gap_packages` optional package for Sage, but there are currently some problems with installing optional packages in the Windows version, which I need to fix... *sigh*
